friends/src/App.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import './App.css';
+import FriendsList from './Components/FriendsList';
+import axios from 'axios';
+import styled from 'styled-components';
+
+const Div = styled.div`
+    background: #e2e1e0;
+    height: 100%;
+`;
+
+class App extends Component {
+    constructor() {
+        super();
+        this.state = {
+            friends: []
+        };
+    }
+
+    componentDidMount() {
+        axios
+            .get('http://localhost:5000/friends')
+            .then(response => {
+                console.log(response);
+                this.setState({ friends: response.data });
+            })
+            .catch(err => {
+                console.log(err);
+            });
+    }
+
+    addNewFriend = e => {
+        e.preventDefault()
+        let send = {
+            name: e.target[0].value,
+            age: e.target[1].value,
+            email: e.target[2].value
```

Comment:
This is interesting; assuming this works, you should be able to access the `name` property too. For example:
```javascript
const send = e.target.slice(0, 3)
  .reduce((obj, input) => ({
    ...obj, 
    [input.name]: input.value
  }), {});
```
or, perhaps cleaner:
```javascript
const send = e.target.slice(0, 3)
  .reduce((obj, { name, value }) => (
    { ...obj, [name]: value }
  ), {});
```